tests/test_api.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from httpx import AsyncClient
+
```

Comment:
v testech chybi:

1. testy OK a funguji spravne.
2. Možná by bylo dobré ověřit obsah odpovědi, když neexistuje žádná odpověď z chatu.